### DISCUSSION TITLE
cpu affinity

### DISCUSSION THREAD

**Discussion by da-line** [axboe/liburing#857] [Q&A]
First of all high respect and thank you for Jens Axboe 
for his great input in the linux kernel.

io_uring does not seem to respect the kernel isolcpus boot option.
io_uring produces significant load also on isolated cpus.
In constrast the old linux_aio doesn't produce any load on isolated cpus.

Is it possible to get io_uring to respect the isolcpus kernel boot option
or set cpu affinity? io_uring_params.sq_thread_cpu only affects the poll thread?

**Comment by sharpobject** [comment]
It looks like you need to use IORING_REGISTER_IOWQ_AFF to choose where the iou-wrk threads will be.
